lib/web_ui/test/engine/semantics/semantics_tester.dart

```diff
@@ -368,13 +368,13 @@ void expectSemanticsTree(String semanticsHtml) {
 
 /// Finds the first HTML element in the semantics tree used for scrolling.
 html.Element? findScrollable() {
-  return appHostNode.querySelectorAll('flt-semantics').cast<html.Element?>().firstWhere(
-        (html.Element? element) =>
-            element!.style.overflow == 'hidden' ||
-            element.style.overflowY == 'scroll' ||
-            element.style.overflowX == 'scroll',
-        orElse: () => null,
-      );
+  return appHostNode.querySelectorAll('flt-semantics').cast<html.Element>().firstWhereOrNull(
```

Comment:
you can drop the `cast`, too!

Reply:
I can't because `querySelectorAll` returns `Node`, not `Element.`

Reply:
Hold on a sec, that's a bug in our `HostNode` class. Stand by!

Reply:
Oh! On `Node`, not document. Ack.

Reply:
Done!